example/README.md

```diff
@@ -1,15 +1,22 @@
 # Amplify Flutter Example
 
-Sample application for getting started with the Amplify Flutter Library
-
-## Getting Started
+Sample flutter app for getting started with the Amplify Flutter Library.  This example uses the Auth, Analytics, and Storage components of the Flutter library. 
 
 Please check out our docs here:
 https://docs.amplify.aws/start/q/integration/flutter
 
-!!! IMPORTANT !!!
+# TL;DR 
+
+**The app will not compile.  You must configure 'amplifyconfiguration.dart' using Amplify CLI.**
+
```

Comment:
I don't think we should say this.  The phrase 'The app will not compile' makes it sound like the app  will NEVER compile. Also, they shouldn't have to use the CLI. Customers can (and many do) create their own config files independent of the CLI.

Reply:
Alright I will remove this section.